core/services/job/kv_orm_test.go

```diff
@@ -34,7 +34,7 @@ func TestJobKVStore(t *testing.T) {
 
 	jobID := int32(1337)
 	kvStore := job.NewKVStore(jobID, db, config.Database(), lggr)
-	jobORM := NewTestORM(t, db, pipelineORM, bridgesORM, cltest.NewKeyStore(t, db, config.Database()), config.Database())
+	jobORM := NewTestORM(t, db, pipelineORM, bridgesORM, cltest.NewKeyStore(t, db), config.Database())
```

Comment:
Is this right? `config.Database()`?

Reply:
What do you mean? Can only remove one for now. The other will come later.